src/whatsnew/2.2.rst

```diff
@@ -27,30 +27,173 @@ Upgrade Notes
 
 .. rst-class:: open
 
-* Item 1
-
-* Item 2
+* TODO WORDING: Recommend increasing PBKDF2 / bcrypt work factor and auth_plugins to
+  couch_replicator_auth_session,couch_replicator_auth_noop. Do not do this, if an older 
+  version of CouchDB is replicating TO this instance/cluster. Replicating FROM older
+  versions is fine. A future version will make this a default.
+
+* :ghissue:`820`, :ghissue:`1032`: Multiple queries can now be made at the
+  ``POST /{db}/_all_docs/queries``, ``POST /{db}/_design_docs/queries`` and
+  ``POST /{db}/_local_docs/queries`` endpoints. Also, a new endpoint
+  ``POST /{db}/_design/{ddoc}/_view/{view}/queries`` has been introduced to replace
+  the ``?queries`` parameter formerly provided for making multiple queries to a view.
+  The old ``?queries`` parameter *is now deprecated and will be removed in a future
+  release of CouchDB.*
+
+* The maximum http request limit, which had been lowered in 2.1.0, has been re-raised
+  to a 4GB limit for now. (:ghissue:`1446`). Ongoing discussion about the path forward
+  for future releases is available in :ghissue:`1200` and :ghissue:`1253`.
+
+* :ghissue:`1153`: The CouchDB replicator can now make use of the ``/_session`` endpoint
+  rather than relying entirely on HTTP basic authentication headers. This can greatly
+  improve replication performance. We encourage you to upgrade any nodes or clusters that
+  regularly act as replication clients to use this new feature, which is enabled by
+  default (:ghissue:`1462`).
```

Comment:
N.B. This PR has not yet merged - if we decide not to push it in 2.2.0, I will update this to reflect that reality.